docs/developer.mdx

```diff
@@ -23,6 +23,7 @@ Whether you're building a custom-branded storefront or looking for ways to exten
 - [**Checkout**](developer/checkout.mdx): the checkout flow.
 - [**Users**](developer/users.mdx): creating and managing customer accounts.
 - [**Metadata**](developer/metadata.mdx): store additional information about items.
+- [**Importing and exporting**](developer/export-products.mdx): import and export data.
```

Comment:
It says import and export but only one direction is currently supported.

Reply:
That's true. So maybe, for now, it should be just exporting.

Reply:
Yeah, let's call it "exporting data".